pkg/packagekit/updates.scss

```diff
@@ -271,8 +271,8 @@ div.changelog {
 }
 
 .flow-list {
+    margin: 0 1rem 1rem;
```

Comment:
flow list is direct child of pf-c-table__expandable-row-content. So by adding left and right margin there, you basically add to pf-c-table__expandable-row-content. So if the expandable rows needs margins, that change should be in cockpit-components-listing-panel.scss and not specifically here. What do you think?

Reply:
I took a look again, this margin is wrong. It creates a difference spacing for the expanded content for this list compared to the list above, and since there is already margin on td this is not needed.

If this is removed I am fine with the rest of the changes.